tests/acceptance/features/bootstrap/WebDavLockingContext.php

```diff
@@ -742,5 +837,6 @@ public function before(BeforeScenarioScope $scope) {
 		// Get all the contexts you need in this context
 		$this->featureContext = $environment->getContext('FeatureContext');
 		$this->publicWebDavContext = $environment->getContext('PublicWebDavContext');
+		$this->spacesContext = $environment->getContext('SpacesContext');
```

Comment:
either we have to conditionally use `$this->spacesContext` or move spaces related steps to another context file (e.g. Spaces context itself). Reva will complain about it